### ISSUE TITLE
Providing `instant_roll` and `booked_roll`

### ISSUE THREAD

**Issue by cusma** [cusma/algo-dices#2]
> I see. I think I will take the best of two worlds in a future PR, refactoring the methods as follows:
>
> - Turning `dice_rolls` into an internal subroutine;
> - Provide a `instant_roll`, which takes the round a player want as method input;
> - Provide a `booked_roll`, which takes the round a player as previously committed to.

_Originally posted by @cusma in https://github.com/cusma/algo-dices/pull/1#discussion_r1055876318_

**Comment by fabrice102** [comment]
Yes, that's a good option. It is also important to make it clear in the README and contract metadata that the contract should only be called once for a given round, using either `instant_roll` or `booked_roll` (even for a different set of dices).
With a counter-example.
